src/chart/speclayout/tablechart.h

```diff
@@ -27,10 +27,11 @@ void TableChart::setupVector(std::vector<Item> &items,
 		if (item.enabled) size++;
 
 	auto rowsize =
-	    static_cast<size_t>(singleColumn ? 1.0 : ceil(sqrt(size)));
+	    static_cast<ssize_t>(singleColumn ? 1.0 : ceil(sqrt(size)));
```

Comment:
why ssize_t?

Reply:
because `std::ldiv` only works with signed integers, and this variable is not used as `size_t`, only as `ssize_t` (and `double`).

Reply:
note: size variable is only an integer, so it's square root must be smaller than ssize_t max always, so there are no over/underflow anywhere.